### PR TITLE
Update messaging library, use apache commons logger

### PR DESCRIPTION
**Pull request by ao508**
Signed-off-by: Angelica Ochoa <15623749+ao508@users.noreply.github.com>

### CODE REVIEW

File: src/main/java/org/mskcc/cmo/publisher/pipeline/MetaDbFilePublisherReader.java [hunk 1]
```diff
@@ -28,7 +29,7 @@
 
     private List<Map<String, String>> messagesToPublish;
 
-    private Logger LOG = Logger.getLogger(MetaDbFilePublisherReader.class);
+    private static final Log LOG = LogFactory.getLog(MetaDbFilePublisherReader.class);
```
Comment (divyamadala30) [inline src/main/java/org/mskcc/cmo/publisher/pipeline/MetaDbFilePublisherReader.java:32]: Should this log be static?

Comment (divyamadala30) [reply src/main/java/org/mskcc/cmo/publisher/pipeline/MetaDbFilePublisherReader.java:32]: Also do we still need log4j properties file? Can i go ahead and remove it?

**Review by divyamadala30 — COMMENTED** [review]
Left a couple of questions as comments

File: src/main/java/org/mskcc/cmo/publisher/pipeline/MetaDbFilePublisherReader.java (hunk 1 above)
Comment (ao508) [reply src/main/java/org/mskcc/cmo/publisher/pipeline/MetaDbFilePublisherReader.java:32]: @divyamadala30 oh yeah good catch, thanks for reminding me

**Pull request merged**